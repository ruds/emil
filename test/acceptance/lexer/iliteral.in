123
123i
-123
-123I
111_222_333_444_555_666_777_888
111_222_333_444_555_666_777_888i
(*xx*)
0123i
0X123i
0o123i
0b1011i
0123
0X123
0o123
0b1011
0xcafed00di
0xCAFED00Di
0xcaFE_d00Di
0xcaFE_d00D
12CAi
(*xx*)
12CA
(*xx*)
0xCAG0
(*xx*)
0xCAG0i
(*xx*)
0o338
(*xx*)
0o338i
(*xx*)
0b2
(*xx*)
0b2i
(*xx*)
