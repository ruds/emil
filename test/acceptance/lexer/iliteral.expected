0001 ILITERAL 123 123
0002 ILITERAL 123i 123i
0003 ILITERAL -123 -123
0004 ILITERAL -123i -123I
0005 ILITERAL 111222333444555666777888 111_222_333_444_555_666_777_888
0006 ERROR
0008 ILITERAL 123i 0123i
0009 ILITERAL 291i 0X123i
0010 ILITERAL 83i 0o123i
0011 ILITERAL 11i 0b1011i
0012 ILITERAL 0123 0123
0013 ILITERAL 0x123 0X123
0014 ILITERAL 0o123 0o123
0015 ILITERAL 0b1011 0b1011
0016 ILITERAL 3405697037i 0xcafed00di
0017 ILITERAL 3405697037i 0xCAFED00Di
0018 ILITERAL 3405697037i 0xcaFE_d00Di
0019 ILITERAL 0xcaFEd00D 0xcaFE_d00D
0020 ERROR
0022 ERROR
0024 ERROR
0026 ERROR
0028 ERROR
0030 ERROR
0032 ERROR
0034 ERROR
0036 END_OF_FILE
